REQUIRE

```diff
@@ -1,4 +1,4 @@
-julia 0.5
+julia 1.0
```

Comment:
Let's change this to `julia 0.7`, which is the most useful version for upgrading other packages (you get deprecation warnings on 0.7, not on 1.0, but it's otherwise "mostly" compatible with 1.0---"mostly" meaning "is compatible except where being compatible with 0.6 makes that impossible").